Create joachim-leonfellner.json

nor0x opened the pull request:
# #WeAreDonet Creator Submission Guidelines

* [x] Check all points before submitting the PR.

* [x] By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* [x] Make sure to follow our [Code of Conduct](https://github.com/wearedotnet/dotnet-creator-cards/blob/master/CODE_OF_CONDUCT.md) 

* [x] We do not accept submissions with the sole intention of promoting a product or company.

* [x]  We add you as a creator, not your company. 

* [x]  The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* [x] You must have published at least three post (long form content) since 2022 on any of your channels. 

* [x] There must be three content pieces in English. 

* [x] There needs to be more than tweets. 

* [x] Longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* [x] Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* [x] Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* [x] If you do not submit an RSS or Atom URL. We cannot consider your feed for the upcoming aggregation feature. 

## Content Guideline 

* [x] If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* [x] We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* [x] You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

DeeJayTC reviewed (changes requested):
Hey @johniwasz 

We only accept projects in form of direct link to public Github Repositories as this is all about being  open, open source and supporting the community. Can  you update this please to point to the projects itself?

nor0x commented:
hi @DeeJayTC 👋

thanks for the review - must have overread that. I fixed in in my latest commit

nor0x commented:
Hi @DeeJayTC 
i have just updated my card with the latest changes from the _creatorTemplate.json and added a .NET specific feed. Do you need anything else from my side?

Thank you 🙂

DeeJayTC commented:
Nope, thanks! 

Ill merge that later today, the split between dotnet and personal feed is not fully rolled out yet but prolly later today
